packages/host/app/components/card-prerender.gts

```diff
@@ -77,6 +78,17 @@ export default class CardPrerender extends Component {
     );
   }
 
+  private warmUpModuleCache = dropTask(async () => {
+    await this.loaderService.loader.import(`${baseRealm.url}card-api`);
+    await this.loaderService.loader.import(`${baseRealm.url}catalog-entry`);
+    await this.loaderService.loader.import(`${baseRealm.url}card-ref`);
+    await this.loaderService.loader.import(`${baseRealm.url}boolean`);
+    await this.loaderService.loader.import(`${baseRealm.url}date`);
+    await this.loaderService.loader.import(`${baseRealm.url}datetime`);
+    await this.loaderService.loader.import(`${baseRealm.url}integer`);
+    await this.loaderService.loader.import(`${baseRealm.url}string`);
```

Comment:
Is there a way to source this list of card types from somewhere so that it doesn't need to be maintained manually?

Reply:
one idea is to recursively get all the directory listings from the base realm after it starts up and issue imports for all the executable files that we discover